logging/upstream_log.go

```diff
@@ -61,15 +60,6 @@ func (u *UpstreamLog) RoundTrip(req *http.Request) (*http.Response, error) {
 		requestFields["bytes"] = req.ContentLength
 	}
 
-	path := &url.URL{
```

Comment:
Looks like the `path` log field also contained a query. With the fix this looks no longer the case. I'm not sure what the intended behaviour is.

Reply:
The `path` has not to contain the `query` string. The `query` is a part of the `url` field in the log.